docs/source/notebooks/model_comparison.ipynb

```diff
@@ -1,25 +1,34 @@
 {
```

Comment:
<p>Maybe use Latex ordered list formatting for more beautiful display?</p><p> </p><p>0) The index <strong>is</strong> <s>are</s> the names of the models taken from the keys of the dictionary passed to <code>compare(.)</code>. </p><p>1) <strong>rank</strong>, the ranking <strong>of</strong> <s>on</s> the models starting from 0 (best model) to the number of models. </p><p>2) <strong>loo</strong>, the values of LOO (or WAIC). (needs a space after LOO)</p><p>5) <strong>weight</strong>, the weights assigned to each model. These weights can be loosely interpreted as the probability of each model <strong>being true</strong> (among the compared models) given the data</p><p>9)... Other options are deviance -- this is the log-score multiplied by -2 (this reverts the order: a <strong>lower</strong> <s>higher</s> LOO/WAIC will be better) -- and negative-log -- this is the log-score multiplied by -1 (as with the deviance scale, a lower value is better).</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>